### PR TITLE
HAWQ-1493. Integrate Ranger lookup JAAS configuration in ranger-admin…

### PR DESCRIPTION
**Pull request by interma**
… plugin jar

### CODE REVIEW

**Comment by interma** [comment]
I was going to use `setProperty("java.security.auth.login.config", conf_file)`, but it not works.

After refer some ranger code, it seems ranger do some hacks by org.apache.ranger.audit.utils.InMemoryJAASConfiguration, so I use the same class as Ranger does.

Tested on my test vagrant cluster, passed.

File: ranger-plugin/admin-plugin/src/main/java/org/apache/hawq/ranger/service/HawqClient.java [hunk 1]
```diff
@@ -74,7 +72,7 @@
     private static final String DEFAULT_DATABASE = "postgres";
     private static final String DEFAULT_DATABASE_TEMPLATE = "DBTOBEREPLACEDINJDBCURL";
     private static final String JDBC_DRIVER_CLASS = "org.postgresql.Driver";
-
+    private static final String jaasApplicationName = "pgjdbc";
```
Comment (linwen) [inline ranger-plugin/admin-plugin/src/main/java/org/apache/hawq/ranger/service/HawqClient.java]: Suggest use all capital letters for static final variable, follow the name convention above, like XXX_XXX_XXX.

File: ranger-plugin/admin-plugin/src/main/java/org/apache/hawq/ranger/service/HawqClient.java [hunk 2]
```diff
@@ -90,6 +88,8 @@
     public HawqClient(String serviceName, Map<String, String> connectionProperties) throws Exception {
         super(serviceName, connectionProperties);
         this.connectionProperties = connectionProperties;
+
```
Comment (linwen) [inline ranger-plugin/admin-plugin/src/main/java/org/apache/hawq/ranger/service/HawqClient.java]: two useless empty lines.

**Comment by zhangh43** [comment]
+1

File: ranger-plugin/admin-plugin/src/main/java/org/apache/hawq/ranger/service/HawqClient.java (hunk 1 above)
Comment (interma) [inline ranger-plugin/admin-plugin/src/main/java/org/apache/hawq/ranger/service/HawqClient.java]: Thanks, I will fix them.

**Comment by linwen** [comment]
LGTM

**Comment by linwen** [comment]
This PR has been merged into master. Please close it. Thanks!

**Comment by interma** [comment]
merged

**Pull request closed without merging**